internal/gapicgen/generator/config.go

```diff
@@ -42,6 +42,10 @@ type microgenConfig struct {
 	// useful if a client needs to be deprecated, but retained in the repo
 	// metadata.
 	stopGeneration bool
+
+	// metadata is used to toggle generation of the gapic_metadata.json file
+	// for the client library.
+	metadata bool
```

Comment:
When is this not set to true? Should we reverse the var so by default it does not need to be set for every bit of config?

Reply:
Good idea. idk of a case where it shouldn't be enabled.

Reply:
Done!